api/src/test/java/com/nicknathanjustin/streamercontracts/contracts/ExpiredContractsSqsHandlerTest.java

```diff
@@ -0,0 +1,56 @@
+package com.nicknathanjustin.streamercontracts.contracts;
```

Comment:
Out of curiosity, how are you getting your code coverage number?

Reply:
Intellij lets us run unit tests 1 of 3 ways:
- run 
- debug
- run with coverage. 

The last option pops up a useful UI to show coverage statistics.

Reply:
Gotcha, I'll have to give this a try at some point.